### ISSUE TITLE
Typing issues using autocomplete

### ISSUE THREAD

**Issue by Evil-Dragon** [DominikDoom/a1111-sd-webui-tagcomplete#42]
I've had to completely remove this since it's causing way too many issues with typing anything into text2img. It basically made me think my keyboard was broken since almost every word i typed would miss letters or miss spaces out. Not sure what is causing it but after removing it completely it has gone back to the way it was.

I really like this addon for A111 and it's been really useful to me in generating prompts but I wish we could perhaps choose what parts of the script are used. For example to be able to disable wildcards/embedding/etc so that the user can pick what they want. For example, i don't really use wildcards/embeddings most of the time if at all but I like using the danbooru/e621 auto complete tags and it gives me ideas when using it that I might not have thought about.

Obviously this is your project and I don't want to come across as being one of those users that demands things every day but I wanted to report what i've found.

Using https://github.com/DominikDoom/a1111-sd-webui-tagcomplete/releases/tag/1.7.3
Firefox
Windows 10

**Comment by Evil-Dragon** [comment]
I'm pretty sure this is also related to https://github.com/DominikDoom/a1111-sd-webui-tagcomplete/issues/40 as well

**Comment by DominikDoom** [comment]
Yeah missing letters sounds like the problem in #40, which is still a WIP due to me not being able to reproduce it.

> I really like this addon for A111 and it's been really useful to me in generating prompts but I wish we could perhaps choose what parts of the script are used. For example to be able to disable wildcards/embedding/etc so that the user can pick what they want. For example, i don't really use wildcards/embeddings most of the time if at all but I like using the danbooru/e621 auto complete tags and it gives me ideas when using it that I might not have thought about.

I would suggest reading the README, the script is pretty configurable and you can turn both of those off.
https://github.com/DominikDoom/a1111-sd-webui-tagcomplete#config

**Comment by Evil-Dragon** [comment]
Sorry I must have missed those options to turn them off. Sometimes I update things without even looking at the config.

**Comment by DominikDoom** [comment]
No worries. Since we are already discussing the broken typing part in #40, I'll close this one to not have duplicates.
